src/summarizedexperiment/BaseSE.py

```diff
@@ -536,3 +543,62 @@ def toAnnData(
         )
 
         return obj
+
+    def combineCols(self, *experiments: "BaseSE", useNames: bool = True) -> "BaseSE":
+        """A more flexible version of `cbind`. Permits differences in the number and identity of rows,
+        differences in `colData` fields, and even differences in the available `assays` among
+        `SummarizedExperiment` objects being combined.
+
+        Only considering RNA-Seq experiments for now.
+
+        Args:
+            experiments ("BaseSE"): `SummarizedExperiment` objects to concatenate.
+            useNames (bool):
+                If `True`, then each input `SummarizedExperiment` must have non-null, non-duplicated row names.
+                The row names of the resultant `SummarizedExperiment` object will be the union of the row
+                names across all input objects.
+                If `False`, then each input `SummarizedExperiment` object must have the same number of rows.
+                The row names of the resultant `SummarizedExperiment` object will simply be the row names of
+                the first `SummarizedExperiment`.
+
+        Raises:
+            TypeError:
+                - if any of the provided objects are not "SummarizedExperiment".
+            ValueError:
+                - if there are null or duplicated row names (useNames=True)
+                - if all objects do not have the same number of rows (useNames=False)
+
+        Returns:
+            BaseSE: combined `SummarizedExperiment` object.
+        """
+
+        if len(experiments) == 1 and isinstance(experiments[0], list):
```

Comment:
trying to understand this, why would `experiments[0]` be a list?

are we trying to fix for `se.combineCols([se2, se3])`?

Reply:
yep, that's exactly why

Reply:
we should not be compensating for users mistakes :) 

i'm gonna modify so that it throws an error

Reply:
do we not want to allow the user to also pass in a list of SEs?